modules/admin-ui-picard/app/src/thunks/eventDetailsThunks.js

```diff
@@ -19,6 +21,54 @@ const getHttpHeaders = () => {
     };
 }
 
+export const saveAccessPolicies = (eventId, policies) => async (dispatch) => {
+
+    let headers = getHttpHeaders();
+
+    let data = new URLSearchParams();
+    data.append("acl", JSON.stringify(
+        {
+            acl: {
+                ace: policies
+            }
+        })
+    );
+    data.append("override", true);
+
+    return axios.post(`admin-ng/event/${eventId}/access`, data.toString(), headers)
+        .then(response => {
+            console.log(response);
+            dispatch(addNotification('info', 'SAVED_ACL_RULES', 5, null, NOTIFICATION_CONTEXT));
+            return true;
+        })
+        .catch(response => {
+            console.log(response);
+            dispatch(addNotification('error', 'ACL_NOT_SAVED', 5, null, NOTIFICATION_CONTEXT));
+            return false;
+        });
+}
+
+export const fetchAccessPolicies = (eventId) => async (dispatch) => {
+    try {
+        const policyData = await axios.get(`admin-ng/event/${eventId}/access.json`);
+        let policies = await policyData.data;
+
+        dispatch(loadEventPoliciesSuccess(policies));
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+export const fetchHasActiveTransactions = (eventId) => async (dispatch) => {
```

Comment:
you don't need dispatch here, because you don't make any redux state changes in this method